models/fishnet.py

```diff
@@ -0,0 +1,244 @@
+import torch
+from torch import nn
+from .blocks.bottleneck import Bottleneck
+
+'''
+FishNet
+Author: Shuyang Sun
+'''
+
+__all__ = ['fish']
+
+
+class Fish(nn.Module):

```

Comment:
FishNet

Reply:
there is already FishNet class in `fishnet.py`